dd-sdk-android/src/main/kotlin/com/datadog/android/rum/RumMonitor.kt

```diff
@@ -181,7 +181,7 @@ interface RumMonitor {
          */
         fun build(): RumMonitor {
             return if (!RumFeature.isInitialized()) {
-                devLogger.e(Datadog.MESSAGE_NOT_INITIALIZED)
+                devLogger.e(Datadog.MESSAGE_NOT_INITIALIZED_WITH_APP_ID)
```

Comment:
This is the wrong error message here. The `NoOpRumMonitor` is used when RUM is disabled in the DatadogConfig (`setRumEnabled(false)`). You should create a (local constant) message that reads: 
> You're trying to create a RumMonitor instance, but the RUM feature was disabled in your DatadogConfig. No RUM data will be sent.

Reply:
Yeah I think I am going to get rid of those messages in the Datadog class as they are becoming obsolete